public/content/documentation/web/form/help-hint-error.md

```diff
@@ -0,0 +1,193 @@
+## General Notes
+
+How to test a hint, help, or error
+
+## Condensed
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+How to test a hint, help, or error
+
+1. Test keyboard only, then screen reader + keyboard actions
+
+   - Tab: The input comes into focus
+
+2. Test mobile screenreader gestures
+
+   - Swipe: The input's name is read. Then the hint, help or error text is read
+
+3. Listen to screenreader output on all devices
+
+   - Name: After the input name, role and state is read. Then the hint, help or error is read
```

Comment:
This reads a little odd. Maybe change to "...After the input name, the role and state is read."

So add "the" before "role.